cli/src/main/java/bio/terra/pfb/JavaPfbCommand.java

```diff
@@ -21,19 +52,69 @@ static int executeCommand(String[] args) {
 
   @Override
   public void run() {
-    System.out.println("PFB RUN");
+    Library library = new Library(new PfbReader());
+    logger.info("PFB RUN");
+    if (command != null && option != null) {
+      if (command.equals(SHOW)) {
+        switch (option) {
+          case SCHEMA:
+            logger.info("Show schema for file path: {}", filePath);
```

Comment:
It's up to you, but I don't think `logger` is the correct tool for producing output from a command line tool, unless the you want the output to look like a log file.

Reply:
I'm a little torn here. 
I do like using the logback appender since it's a standard across our java projects and it has helpful error message formatting built-in.
It does seem like we can format the logs however desired with the logback yaml. 

But, PrintStream or System.out should also both be pretty straightforward.

Reply:
I think either could work. IMO the logging API's output isn't good CLI output in that it has timestamps and other things that a usual CLI user wouldn't be interested in (which can be customized) . It also requires configuration and is overall less flexible than using a `PrintStream`. In a spring boot app `System.out` use is discouraged but I think in a CLI app it makes sense.

It's true that the log APIs make it easy to include a stack trace in the output, although I think for a CLI a "exceptional" exception shouldn't be caught by the app code. The top-level exception handler will catch it an print the error. In the case where an exception is expected I think you'd want to catch it and print out a custom message anyway.

On the other hand, I've seen long running CLIs that are multithreaded which use a logger for their output. If the CLI is running long enough or needs lots of diagnostic output then it could be useful.

Reply:
Yeah, I have had to constantly ignore warnings from sonarcloud about using system.out. 

Thinking back to the goal of this CLI is to more-or-less replicate the output from pyPFB. They only output the result of the command. This has been useful, because I've been able to directly write the output to JSON files/etc. 
So, I think I will go with your suggestion to go back to `System.out`.

Reply:
I know you can customize the sonar warnings in the project dashboard (although I haven't done this and don't know exactly how it's done) to silence the `System.out` warning. Or you can change your code to use `PrintStream` and isolate the reference to `System.out` in a single place and then use`@SuppressWarnings` to disable the warning in that one place. For an example of this see https://github.com/DataBiosphere/terra-data-catalog/blob/main/common/src/main/java/bio/terra/catalog/common/RequestContextCopier.java#L48